src/main/java/com/google/firebase/auth/multitenancy/ListTenantsPage.java

```diff
@@ -198,9 +197,9 @@ ListTenantsResponse fetch(int maxResults, String pageToken)
 
   static class DefaultTenantSource implements TenantSource {
 
-    private final FirebaseUserManager userManager;
+    private final FirebaseTenantClient userManager;
```

Comment:
Rename this variable to `tenantClient`. It's not a FirebaseUserManager anywmore!

Reply:
Done